action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
     description: Which check run conclusion type to use when annotations are created ("neutral" or "failure" are most common)
     required: false
     default: "neutral"
+  additionalGems:
+    description: Comma-separated list of other gems that your RuboCop setup depends on, in addition to gems starting with "rubocop", which are installed by default
+    required: false
+    default: "standard"
```

Comment:
I only added this as the default because that should prevent this from being a breaking change.  With the changes involved in this PR, if we decide to move forward, I'd suggest removing this default and releasing as "v1.0" with a note that anyone using `standard` will need to add it via this input.